.azure/bicep/decsys/prod.bicepparam

```diff
@@ -0,0 +1,11 @@
+using '../main.bicep'
+
+param location = 'uksouth'
```

Comment:
we don't need to override location - the default (matching RG) is fine for this environment

```suggestion
```